docs/parse_argparse.py

```diff
@@ -0,0 +1,315 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+COPYRIGHT
+Copyright 2020 Ian J. Miller, Evan R. Rees, Kyle Wolf, Siddharth Uppal,
+Shaurya Chanana, Izaak Miller, Jason C. Kwan
+
+This file is part of Autometa.
+
+Autometa is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Autometa is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with Autometa. If not, see <http://www.gnu.org/licenses/>.
+COPYRIGHT
+
+Script used to parse the arparse block of all the autometa scripts, copy them to a new
+file and run the help text (--help) from there.
+Used as an alternative to sphinxcontrib-programoutput which used all the heavy dependencies
+preventing the intergration with readthedocs.
+"""
+
+import glob
+import os
+import re
+import shutil
+import subprocess
+import tempfile
+import textwrap
+
+
+def get_argparse_block(fpath):
+    """
+    Copies the argparse block of each script in a temporary file
+
+    Parameters
+    ----------
+    fpath : string
+        file path that needs to be documented,
+        e.g. '</path/to/autometa/script.py>'
+
+    Returns
+    -------
+    string
+        path to temporary file that has the argparse block
+        e.g. '</path/to/tmp/file>'
+    """
+    writing = False
+    outlines = ""
+    with open(fpath) as fh:
+        for line in fh:
+            line = line.strip()
+            if line == "import argparse":
+                writing = True
+                imports = ["import os",
+                           "import multiprocessing as mp", ""]
+                # the blank "" is needed to give a new line else error
```

Comment:
If there is no `""` then the next line is written on the same line as `import multiprocessing as mp`. The output will be `import multiprocessing as mpimport argparse` 
I can make it `"\n"` so it will be more intuitive.